Source/AtomicJS/Javascript/JSAtomic.cpp

```diff
@@ -237,6 +237,29 @@ static int js_atomic_ScriptEvent(duk_context* ctx)
     return 1;
 }
 
+static int js_atomic_ScriptEventData(duk_context* ctx)
+{
+    if (duk_get_top(ctx) != 2 || !duk_is_string(ctx, 0))
+    {
+        duk_push_string(ctx, "Atomic.ScriptEventData(eventType:string, data?:Object); - passed invalid parameters");
+        duk_throw(ctx);
+        return 0;
```

Comment:
We could definitely use some more debugging with throws native side for JS stack, especially for debug builds, just noting